python/python_fastapi_server/routers/ogcapi.py

```diff
@@ -122,6 +122,9 @@ async def handle_ogc_api_root(req: Request, f: str = "json"):
     landing_page = LandingPage(title="ogcapi",
                                description="ADAGUC OGCAPI-Features server",
                                links=links)
+
+    response.headers['cache-control']="max-age=18"
```

Comment:
Do we want to hard-code this?

There is also a TTLCache in this file that doesn't seem to be used.

Reply:
Removed

Reply:
Not yet.